index.js

```diff
@@ -1,302 +1,293 @@
 class Say {
 
-  static append(val) {
-    return new Say().append(val);
-  }
-
-  static sentence(val) {
-    return new Say().sentence(val);
-  }
-  constructor() {
-    this.str= "";
-  }
-
-  append(val) {
-    this.str += val;
-    return this;
-  }
-
-  newline() {
-    return this.append("\n");
-  }
-
-  paragraph() {
-    return this.newline().newline();
-  }
-
-  sentence(val) {
-    return this.append(" ").append(val);
-  }
-
-  toString() {
-    return this.str.trim();
-  }
+	static append(val) {
+		return new Say().append(val);
+	}
+
+	static sentence(val) {
+		return new Say().sentence(val);
+	}
+
+	constructor() {
+		this.str = '';
+	}
+
+	append(val) {
+		this.str += val;
+		return this;
+	}
+
+	newline() {
+		return this.append('\n');
+	}
+
+	paragraph() {
+		return this.newline().newline();
+	}
+
+	sentence(val) {
+		return this.append(' ').append(val);
+	}
+
+	toString() {
+		return this.str.trim();
+	}
+
 }
 
 class Convo {
 
-  static mockConv() {
-    return {
-      surface:{
-        capabilities:{
-          has:() => true
-        }
-      },
-      available:{
-        surfaces:{
-          capabilities:{
-            has:() => true
-          }
-        }
-      },
-      user:{
-        access:{},
-        storage:{}
-      },
-      contexts:{}
-    }
-  }
-  static setClassMappings(mappings) {
-    this._clsMappings = mappings;
-  }
-
-  static SignIn(obj) {
-    if (this._clsMappings && this._clsMappings['SignIn']) {
-      return this._clsMappings['SignIn'](obj);
-    }
-    return {type:"SignIn", data:obj};
-  }
-  static SimpleResponse(obj) {
-    if (this._clsMappings && this._clsMappings['SimpleResponse']) {
-      return this._clsMappings['SimpleResponse'](obj);
-    }
-    return {type:"SimpleResponse", data:obj};
-  }
-  static NewSurface(obj) {
-    if (this._clsMappings && this._clsMappings['NewSurface']) {
-      return this._clsMappings['NewSurface'](obj);
-    }
-    return {type:"NewSurface", data:obj};
-  }
-  static Image(obj) {
-    if (this._clsMappings && this._clsMappings['Image']) {
-      return this._clsMappings['Image'](obj);
-    }
-    return {type:"Image", data:obj};
-  }
-  static List(obj) {
-    if (this._clsMappings && this._clsMappings['List']) {
-      return this._clsMappings['List'](obj);
-    }
-    return {type:"List", data:obj};
-  }
-  static BasicCard(obj) {
-    if (this._clsMappings && this._clsMappings['BasicCard']) {
-      return this._clsMappings['BasicCard'](obj);
-    }
-    return {type:"BasicCard", data:obj};
-  }
-  static Button(obj) {
-    if (this._clsMappings && this._clsMappings['Button']) {
-      return this._clsMappings['Button'](obj);
-    }
-    return {type:"Button", data:obj};
-  }
-
-  static isPromise(obj) {
```

Comment:
I pulled this outside of the class since it wasn't class specific.  It's added again below